apple/Elements/RNSVGSvgView.m

```diff
@@ -25,22 +25,25 @@ @implementation RNSVGSvgView
 - (instancetype)initWithFrame:(CGRect)frame
 {
     if (self = [super initWithFrame:frame]) {
+#if !TARGET_OS_OSX
+        // TODO: Figure out the proper macOS equivalent
```

Comment:
Do we need this TODO if it's working on Mac?

Reply:
Discussed offline. Since macOS appears to work fine, I'm simply removing this TODO.